lib/formatters/csv.js

```diff
@@ -177,7 +181,29 @@ function toCsvString (rows) {
     .join('\n');
 }
 
+function sortCsv (a, b) {
```

Comment:
This doesn’t actually do any sorting, so something like `compareRows` or `rowComparator` would be a clearer (and more traditional) name.